
namespace Ej4CustomException
{
    internal class Logic
    {

        public static void ThrowCustomException()
        {
            throw new CustomException("Error de prueba custom");
        }
    }
}
